added note to top of page

mking18 opened the pull request:
Jay and Michael - I really wanted to include the note that this page is visual reference only and add a link to the SVG downloads. Figured to mention where designers and developers would find those icons - let me know if this is the right terminology and links to share.

mhewson commented:
I updated this to display with an info banner @mking18 let me know if you would like me to merge this in
<img width="1244" alt="Screen Shot 2021-08-03 at 11 49 41 AM" src="https://user-images.githubusercontent.com/5115458/128069810-6353b1ac-2f1b-40a3-83bc-cc9cdfb3967c.png">

mking18 commented:
Looks good, thanks!

From: mhewson ***@***.***>
Reply-To: rei/rei-cedar-docs ***@***.***>
Date: Tuesday, August 3, 2021 at 11:50 AM
To: rei/rei-cedar-docs ***@***.***>
Cc: Maya Hampton ***@***.***>, Mention ***@***.***>
Subject: Re: [rei/rei-cedar-docs] added note to top of page (#939)

I updated this to display with an info banner @mking18 let me know if you would like me to merge this in — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub, or unsubscribe. Triage notifications ZjQcmQRYFpfptBannerStart
This Message Is From an External Sender
This message came from outside your organization.
ZjQcmQRYFpfptBannerEnd


I updated this to display with an info banner @mking18<https://urldefense.com/v3/__https:/github.com/mking18__;!!M0aTtQ!O06EGAqV0G6qgmQMmaDWFmwtACoOzPrPazTU5I6KFzVIzqSGF8slo4oMdvM$> let me know if you would like me to merge this in
[Screen Shot 2021-08-03 at 11 49 41 AM]<https://urldefense.com/v3/__https:/user-images.githubusercontent.com/5115458/128069810-6353b1ac-2f1b-40a3-83bc-cc9cdfb3967c.png__;!!M0aTtQ!O06EGAqV0G6qgmQMmaDWFmwtACoOzPrPazTU5I6KFzVIzqSGF8slNC0Y_gQ$>

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https:/github.com/rei/rei-cedar-docs/pull/939*issuecomment-892082182__;Iw!!M0aTtQ!O06EGAqV0G6qgmQMmaDWFmwtACoOzPrPazTU5I6KFzVIzqSGF8sluRZ50DI$>, or unsubscribe<https://urldefense.com/v3/__https:/github.com/notifications/unsubscribe-auth/AKIT7TL75V2T537SOKDEB73T3A3ARANCNFSM5BNQZ6IA__;!!M0aTtQ!O06EGAqV0G6qgmQMmaDWFmwtACoOzPrPazTU5I6KFzVIzqSGF8slkp7GxFM$>.
Triage notifications on the go with GitHub Mobile for iOS<https://urldefense.com/v3/__https:/apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675__;!!M0aTtQ!O06EGAqV0G6qgmQMmaDWFmwtACoOzPrPazTU5I6KFzVIzqSGF8slYqUoUXU$> or Android<https://urldefense.com/v3/__https:/play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email__;!!M0aTtQ!O06EGAqV0G6qgmQMmaDWFmwtACoOzPrPazTU5I6KFzVIzqSGF8slGlOQuQE$>.
